Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.py

```diff
@@ -305,23 +509,28 @@ def route_list_values() -> Response:
                 demisto.debug(err_msg)
                 return Response(err_msg, status=401)
 
-        limit, offset, out_format, query = get_request_args(params)
+        request_args = get_request_args(params)
 
         values = get_outbound_ioc_values(
```

Comment:
With this amount of arguments being passed around, I think the code would be more readable if you extract all of this data into a class such as `OutboundIndicatorsModel`, and just pass it rather than all the arguments.